lib/simple_form/components/input_groups.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Components
+    module InputGroups
+      module Prepends
+        def prepend(wrapper_options = nil)
+          span_tag = content_tag(:span, options[:prepend], class: "input-group-text")
```

Comment:
If the tags and the html classes are customizable it could work with another CSS frameworks as well. Makes sense? 🤔

Reply:
Yes, totally agree! Do you have a suggestion how to prepare these css classes for customization?